generators/ci-cd/templates/travis.yml.ejs

```diff
@@ -44,6 +43,8 @@ env:
     - JHI_DISABLE_WEBPACK_LOGS=true
     - NG_CLI_ANALYTICS="false"
 before_install:
+  - wget https://github.com/sormuras/bach/raw/master/install-jdk.sh
+  - source install-jdk.sh --feature 11
```

Comment:
Please don't edit this file, this is @pascalgrimaud job as build master to decide on the process to install the jdk, please don't use some random script.

Reply:
@PierreBesson : Sorry about this. I should have mentioned this at https://github.com/jhipster/generator-jhipster/pull/10175 where we changed the travis pipeline to use openjdk 11. Later I've realized that perhaps it was better to use the install-jdk script rather than relaying on the default jdk of bionic (which probably can change over time).  

I'll revert this file here; and we'll discuss this in that pull request.

Reply:
@SudharakaP : I'd prefer to be consistent and do the same than our current build (for generator-jhipster). See here https://github.com/jhipster/generator-jhipster/blob/master/.travis.yml#L72-L81
Is it fine for you ?

Reply:
@pascalgrimaud : Yes, I am fine with that. Let me make that change to https://github.com/jhipster/generator-jhipster/pull/10175